test/allowlist_for_publicAPI.json

```diff
@@ -2610,23 +2610,6 @@
     "pipe_split",
     "pipeline"
   ],
-  "torch.distributed.pipelining.PipelineSchedule": [
```

Comment:
do these not need to be re-added? What was their original purpose

Reply:
Those were jail breaks :)